.github/workflows/deploy.sh

```diff
@@ -27,7 +27,7 @@ if [ -n "$selection" ]; then
             branch=main
         ############### FEATURE ###############
         elif [ "$destination" = "feature" ]; then
-            branch=main-iterations
+            branch=main
```

Comment:
are we deleting the feature environment? in which case probably best to totally remove this section

Reply:
I think it is already deleted, can you double check in Azure please?

Reply:
it is deleted so this can be deleted @abhi-markan

Reply:
Thanks @Zainzzkk i've removed `feature` from all deployment related scripts/YML